### ISSUE TITLE
[Critical] Data table has missing or incomplete header cell markup

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#57]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87530 |
| **Checkpoint** | 1.3 Adaptable › Data Tables (1.3.1.b) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Dashboard |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/dashboard> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 10:53 am |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3caccess+@gmail.com |

## Description

The data table has missing or incomplete header cell markup.
The data table under the heading Recent Work Items doesn't have header cells. Since there are columns for Name, Status and Date, this set of data needs table mark up.  
However, the Recent Notification table is more simple and does not need table mark up, but still announces the column numbers which adds extra verbosity.

## Recommendation

Rule : 
Table headers MUST be designated with <th>.

How To Fix: 
Fix this issue by doing ALL of the following for the Recent Work Items:
1. Mark all column header and/or row header cells with a <th> element.
2. Ensure that header cell and data cell relationships are correctly conveyed according to the complexity of the table.

For the Recent Notifications:
1. Add role="presentation" to the <table> tag. 

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/tables/

Background: 
People who are blind cannot see the organizational structure of a table with data arranged in rows and columns with corresponding header cells. In order for screen reader users to understand the logical relationships of data arranged in a table, tables need HTML markup that indicates header cells and data cells and defines their relationship. When tables are marked correctly, screen reader users are able to navigate data tables from cell to cell, in a multi-directional way (up, down, right, left), much like navigating a spreadsheet. As they move from cell to cell, screen readers will read the associated header labels.

## Source Code

```
<table class="table is-hoverable is-fullwidth has-padding is-borderless">
```

## Screenshots

![dashboard_table header](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87530/dashboard_table_header.png)
